### PR TITLE
Add Emacs

### PR DESCRIPTION
**Pull request by 10sr**
:pray: 

- Add environment to evaluate Emacs Lisp scripts
- Homepage: https://www.gnu.org/software/emacs/emacs.html 
- Emacs Lisp Reference: https://www.gnu.org/software/emacs/manual/elisp.html

### CODE REVIEW

File: cattleshed-conf/compilers.py [hunk 1]
```diff
@@ -2099,6 +2099,32 @@ def make_openssl(self):
...
+        emacs_vers = get_generic_versions('emacs', with_head=False)
+        compilers = []
+        for cv in emacs_vers:
+            display_name = 'Emacs'
+            version_command = ['/bin/echo', '{cv}']
+
+            compilers.append(format_value({
+                'name': 'emacs-{cv}',
+                'displayable': True,
+                'language': 'Emacs Lisp Script',
```
Comment (melpon) [inline cattleshed-conf/compilers.py]: The language name is too long, so probably Wandbox UI will be strange. Would you please change it into `'Emacs Lisp'`?

Comment (melpon) [reply cattleshed-conf/compilers.py]: If Emacs Lisp is Lisp, would you please change to `'Lisp'`.

Comment (10sr) [reply cattleshed-conf/compilers.py]: 🙆 I changed to `Lisp`

**Comment by melpon** [comment]
The pull request works perfectly: https://wandbox.org/permlink/oDeT7EvqTKgLc7uA
Thank you for your contribution!

**Comment by 10sr** [comment]
Thanks!!!!

**Pull request merged**